### DISCUSSION TITLE
Unsupported: Zemismart curtain robot.

### DISCUSSION THREAD

**Discussion by ThirstyDursty** [Koenkk/zigbee2mqtt#11826] [Devices]
Bought a Zemismart Curtain Robot. Is unsupported. Not sure I fully understand the documentation to add an unsupported device. But maybe a tutorial somewhere? Or an example Tuya cover that could be used?

TS0601
Model is : ZM85EL-2Z
Zigbee manufacturer: _TZE200_cf1sl3tj

**Comment by UndeRus** [comment]
You can find converter here https://github.com/Koenkk/zigbee2mqtt/issues/11251
